src/huggle_ui/editform.cpp

```diff
@@ -115,8 +115,8 @@ void EditForm::DisplayPreview(QString html)
 
 void EditForm::FinishEdit()
 {
-    UiGeneric::pMessageBox(this, "Edited", "Page was successfuly edited, you can close this form now");
-    this->webView->RenderHtml(":)");
+    UiGeneric::pMessageBox(this, "Edited", "Page was successfuly edited");
+    this->hide();
```

Comment:
you really should do `this->close();` otherwise this will create memory leak, as the form is deleted only when closed, and hide really just hides it.